Exception: python3 titli.py
